src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/ManagedIndexRunner.kt

```diff
@@ -441,4 +466,20 @@ object ManagedIndexRunner : ScheduledJobRunner,
             logger.error("There was an error while trying to update the policy_id ($policyID) setting for $index", e)
         }
     }
+
+    private suspend fun publishErrorNotification(policy: Policy, managedIndexMetaData: ManagedIndexMetaData) {
+        policy.errorNotification?.run {
+            errorNotificationRetryPolicy.retry(logger) {
+                withContext(Dispatchers.IO) {
+                    destination.publish(null, compileTemplate(messageTemplate, managedIndexMetaData))
```

Comment:
Publishing message with no subject? Is it because we have more messenger type notifications?

Reply:
The subject was for "SNS" destination types which we do not currently support in ISM. We might in the future and I'd like to keep the structure/calls the same as the notifications in alerting as we eventually move notifications into its own repository. This will make the integration with that easier as noting differs from alerting.